proctor/tests/test_context_processors.py

```diff
@@ -0,0 +1,19 @@
+from unittest import TestCase
+
+try:
+    from unittest.mock import Mock
+except ImportError:
+    from mock import Mock
```

Comment:
Nitpick: I don't think it's worth doing these try/excepts, since this package is requiring the `mock` package for all installs so this line is always valid. This can be replaced with `unittest.mock` when dropping support for Python 2.7